client/webpack.common.config.js

```diff
@@ -6,12 +6,8 @@ module.exports = {
 
   // the project dir
   context: __dirname,
-  entry: ['./assets/javascripts/App'],
+  entry: ['jquery', 'jquery-ujs', './assets/javascripts/App'],
```

Comment:
So two uses cases:

1. Simple one where we just have one bundle.
2. Slightly more complicated one creating a separate jquery, jquery-ujs

I think the performance of `webpack -w` will be a tiny bit better if we don't put the jquery in there every time.

Plus some rails pages might not need react stuff.

So I'd say we put this in as a good example of handling `jquery-bundle.js` as something in general for JS that needs to be available to both Rails and the other webpack bundles.